boot/bootutil/src/loader.c

```diff
@@ -185,11 +201,17 @@ boot_status_source(void)
              table->bst_magic_scratch   == state_scratch.magic) &&
             (table->bst_copy_done_slot0 == 0    ||
              table->bst_copy_done_slot0 == state_slot0.copy_done)) {
-
-            return table->bst_status_source;
+            uint8_t source = table->bst_status_source;
```

Comment:
How about putting .dir-locals.el in .gitignore, but including a copy of what to put there in the tree and mentioning it from the README?

I've made the changes to this pull request to keep the bootutil code in line with mynewt style. Can we submit patches bringing boot/zephyr into compliance later on, as a separate PR, since they already use tabs?